### PR TITLE
project setup

### PR DESCRIPTION
**Pull request by businesskaniki**
- setting  up the project
- adding all  necessary files
- adding a readme file

### CODE REVIEW

File: src/components/Bookinput.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+function Bookinput() {
+  return (
+    <form>
+      <input type="text" placeholder="book title" />
+      <button type="button">submit</button>
+    </form>
+  );
+}
+
+export default Bookinput;
```
Comment (roshan-bajgain) [inline src/components/Bookinput.jsx]: - [ ] It should take the book data as a prop and renders the title and the author. please implement it

File: src/components/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>Categories</div>
+  );
+}
```
Comment (roshan-bajgain) [inline src/components/Categories.jsx]: - [ ] The categories should display only a button with the text "Check status". Please implement it

File: src/components/Bookinput.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Bookinput() {
+  return (
+    <form>
+      <input type="text" placeholder="book title" />
+      <button type="button">submit</button>
+    </form>
```
Comment (roshan-bajgain) [inline src/components/Bookinput.jsx]: - [ ] The CreateNewBook should render a form with inputs for the title and the author of the book. Please implement it.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @businesskaniki, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi, @businesskaniki  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**